plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile

```diff
@@ -42,23 +42,223 @@ p=. Parallel regions
+States are the central elements of a state machine. A state has to be placed inside a region and must have a name that is unique inside this region. During state machine execution, a state can be active or passive.
+
+A state can have a _behavior_. The behavior specifies which actions are executed. Such actions can be triggered by entering the state, leaving the state, occurrence of events, conditions becoming true, or time passing. The behavior specification is a text inside the state's box and consists of a sequence of so-called _local reactions_.
+
+The state *Countdown* in the following example has three local reactions:
+
+p(#fig_state-example). 
+!(standard-image)images/docu_state_010_example_01.png(State example)!
+
+p=. State example: A simple state machine to launch a rocket. When the *Countdown* state is entered, it sets the counting variable _t_ to 10. As long as that variable's value is greater than 0, each second it gets counted down by 1. The operation _speak_ cares for audible information on how many seconds to go until rocket launch. If _t_ has been counted down to 0 – checked by a transition's _guard condition_ –, the *Countdown* state will become inactive and launches the rocket upon exiting. The countdown can be aborted anytime before launch by a _cancel_ event. (See figure "&quot;Transition example&quot;":#fig_transition-example for a different state machine which solves the same task but puts more emphasis on transitions.)
+
+Figure "&quot;State behavior syntax overview&quot;":#fig_state-syntax-overview shows how the syntax of a state's behavior is defined. It consists of an arbitrary number of _local reactions_, where each local reaction essentially consists of a _reaction trigger_ and a _reaction effect_.
+
+p(#fig_state-syntax-overview). 
+!(standard-image)images/docu_grammar_030_state-overview.png(State syntax overview)!
+
+p=. State behavior syntax overview
+
+h4(#local-reactions). Local reactions
+
+If an "event":#events occurs, it may trigger a so-called _local reaction_ in the active state. It is executed completely inside the active state without leaving it, thus the word "local". The UML calls local reactions "internal transitions".
+
+Each local reaction consists of three parts, as shown by figure "&quot;Local reaction syntax&quot;":#fig_local-reaction-syntax.
+
+p(#fig_local-reaction-syntax). 
+!(standard-image)images/docu_grammar_030_state-localreaction.png(Local reaction syntax)!
+
+p=. Local reaction syntax
+
+* The _reaction trigger_. It specifies the conditions that have to be fulfilled for the local reaction to be executed. It consists of one or more "_event specifications_":#event-specifications and a "_guard condition_":#guard-condition. One of these, i.&nbsp;e. either event specification(s) or guard condition, may be omitted, but not both.
```

Comment:
In the preview, I see an unintended underscore at "reaction trigger_" and "_guard condition". Not sure why.